vela/admin.go

```diff
@@ -254,3 +254,17 @@ func (svc *AdminWorkerService) RegisterToken(hostname string) (*library.Token, *
 
 	return t, resp, err
 }
+
+// GetQueueCreds fetches queue credentials based valid registration token.
+func (svc *AdminWorkerService) GetQueueCreds() (*library.QueueRegistration, *Response, error) {
```

Comment:
does this function belong in its own `QueueService` ? like in `vela/queue.go`

Reply:
you'd have to create that service/file since this is our first queue function we've exposed in the sdk. our pattern tends to try to match the api resource with the "service"